util/topgen/merge.py

```diff
@@ -560,6 +569,25 @@ def amend_alert(top):
                 ip[0]["alert_list"]))
 
 
+def amend_wkup(topcfg: OrderedDict):
```

Comment:
@eunchan 
@msfschaffner 

i had a question as i was looking at `amend_alerts` and others, why do we have to declare `alert_modules` and `interrupt_modules`? This information can basically be found while searching through after merging, do we really need the user to declare them?

Reply:
Intention is to make some of the interrupts float if the top doesn't want to connect it. If you leave interrupt_modules blank or undefined, topgen adds every interrupts to the list.

Reply:
yes IMO. Although i was pushing in the direction of not having any modules for interrupts / wakeups / alerts :)
Btw, in the near future, I will most likely also create a 'reset' category...